### ISSUE TITLE
Installation filed due to some dependencies 

### ISSUE THREAD

**Issue by imontalvomiguel** [carlhuda/janus#619]
At MacOSX Terminal app

```
$ curl -Lo- https://bit.ly/janus-bootstrap | bash
```

```
...
Receiving objects: 100% (123/123), 24.26 KiB | 0 bytes/s, done.
Resolving deltas: 100% (48/48), done.
Checking connectivity... done.
Cloning into 'janus/vim/tools/vim-gitgutter'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Clone of 'git@github.com:airblade/vim-gitgutter.git' into submodule path 'janus/vim/tools/vim-gitgutter' failed
rake install
mkdir -p _backup
mkdir -p _temp
ln -s /Users/miguel/.vim/janus/vim/vimrc /Users/miguel/.vimrc
ln -s /Users/miguel/.vim/janus/vim/gvimrc /Users/miguel/.gvimrc

****************************************
*     Post Installing sampler-pack     *
****************************************

➜  ~
```

**Comment by imontalvomiguel** [comment]
Solved I cloned manually ```vim-gitgutter``` from ```https://github.com/airblade/vim-gitgutter.git``` to the janus folder installation tools ```/Users/miguel/.vim/janus/vim/tools```

Then then ```rake``` at ```~/.vim```

**Comment by kalbasit** [comment]
Thanks, fixed in 049be5eefe5c3e77ec331ce7c4b6201e412e9f70
